esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2014,29 +2016,34 @@ void CFileSprayEx::readAndCheckSpraySourceReq(IEspContext& context, MemoryBuffer
                 continue;
 
             //Parse the file to find/validate possible host/ip and local path.
-            StringBuffer hostInPath, localPath;
-            const char* path = nullptr;
-            if (isHostInSprayPath(file, sourcePlaneReq, dropZone, sourceIPReq, sourceHostEp, localPath, hostInPath))
-                path = localPath.str();
-            else
-                path = file;
-
             //Validate file path.
-            StringBuffer s;
-            if (isAbsolutePath(path))
+            StringBuffer hostInPath, localPath, absPath;
+            const char* path = nullptr;
+            if (isUNCPath(file, sourcePlaneReq, dropZone, sourceIPReq, sourceHostEp, localPath, hostInPath))
             {
+                path = localPath.str();
                 //Based on the tests, the dfuserver only supports the wildcard inside the file name, like '/path/f*'.
                 //The dfuserver throws an error if the wildcard is inside the path, like /p*ath/file.
```

Comment:
I think this comment is now misplaced, it was related to the ScopePermissions and tailing wildcard for a File lookup (which looks at leading scopes). This comment should be moved.

Reply:
I will move this comment back to the place the getDZFileScopePermissions is called. It is here because someone may be wondering what if a wildcard is inside the path and the isPathInPlane returns a false.

Reply:
>. It is here because someone may be wondering what if a wildcard is inside the path and the isPathInPlane returns a false.

suggests needs different comment here also.